json_to_env.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""
+Read JSON file and dump contents to source-able shell script.
+
+Useful for extracting Clowder's $ACG_CONFIG file (usually '/cdapp/cdappconf.json').
```

Comment:
minor nit-pick, it's actually /cdapp/cdappconfig.json

Reply:
Good catch! Everywhere else just uses the env var, but I should fix this docstring to be as accurate as possible.